src/backend/distributed/planner/INSERT_SELECT_PLANNER_README

```diff
@@ -0,0 +1,16 @@
+Citus supports INSERT ... SELECT queries either by pushing down the whole query to the
```

Comment:
I think it would be nice if this was a markdown file (`.md`) so it gets nicely rendered in github. It might also be useful to make one file for the whole planner with different sections so you can easily scroll/search through it. So maybe just call it `planner_readme.md` and add a `## INSERT...SELECT via the coordinator` header.